doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1039 @@
+  * :ref:`ug_matter_gs_adding_cluster` page with instructions on how to use ZAP tool binaries. Before this release, the ZAP tool had to be built from sources.
+  * :ref:`ug_matter_hw_requirements` with updated memory requirement values valid for the |NCS| v2.3.0.
+
+* Fixed:
+
+  * The issue where the connection would time out when attaching to a Wi-Fi access point that requires Wi-Fi Protected Access 3 (WPA3).
+  * The issue where the ``NetworkInterfaces`` attribute of General Diagnostics cluster would return EUI-64 instead of MAC Extended Address for Thread network interfaces.
+
+* Removed support for Android CHIP Tool from the documentation and release artifacts.
+  Moving forward, we recommend using the development tool CHIP Tool for Linux or macOS or mobile applications from publicly available Matter Ecosystems.
+
+See `Matter samples`_ for the list of changes for the Matter samples.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``1.0.0.2`` tag.
+
+The following list summarizes the most important changes inherited from the upstream Matter:
+
+* Added:
+
+  * The initial implementation of Matter's cryptographic operations based on PSA crypto API.
+  * Build-time generation of some Zigbee Cluster Library (ZCL) source files using the :file:`codegen.py` Python script.
+  * An alternative factory reset implementation that erases the entire non-volatile storage flash partition.
+
+* Updated:
+
+  * Basic cluster has been renamed to Basic Information cluster to match the specification.
+
+Thread
+------
+
+* Added:
+
+  * Support for setting the default Thread output power using the :kconfig:option:`OPENTHREAD_DEFAULT_TX_POWER` Kconfig option.
+  * A Thread :ref:`power consumption data <thread_power_consumption>` page.
+
+See `Thread samples`_ for the list of changes for the Thread samples.
+
+Zigbee
+------
+
+* Updated Zigbee Network Co-processor Host package to the new version v2.2.1.
+
+* Fixed an issue where buffer would not be freed at the ZC after a secure rejoin of a ZED.
+
+Enhanced ShockBurst (ESB)
+-------------------------
+
+* Added support for front-end modules.
+  The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
+
+* Updated:
+
+  * Number of PPI/DPPI channels used from three to six.
+  * Events 6 and 7 from the EGU0 instance by assigning them to the ESB module.
+  * The type parameter of the function :c:func:`esb_set_tx_power` to ``int8_t``.
+
+Wi-Fi
+-----
+
+* Added:
+
+  * New sample :ref:`wifi_sr_coex_sample` demonstrating Wi-Fi Bluetooth LE coexistence.
+  * :ref:`ug_wifi` document.
+  * :ref:`lib_wifi_credentials` library to store credentials.
+  * :ref:`wifi_mgmt_ext` library to provide an autoconnect command based on Wi-Fi credentials.
+
+* Removed:
+
+  * nRF7002 revision A support.
```

Comment:
And move this probably after 'Updated' `?
```suggestion
* Removed the support for nRF7002 revision A.
```

Reply:
yes